execution/src/evm.rs

```diff
@@ -220,31 +203,36 @@ impl<'a, R: ExecutionRpc> ProofDB<'a, R> {
     }
 
     fn get_account(&mut self, address: Address, slots: &[H256]) -> Result<Account> {
-        /*
-        let execution = self.execution.clone();
-        let payload = self.current_payload.clone();
-        let slots = slots.to_owned();
-
-        let handle = tokio::task::spawn_blocking(move || {
-            block_on(execution.get_account(&address, Some(&slots), &payload))
-        });
-
-        block_on(handle)?
-        */
+        #[cfg(not(target_arch = "wasm32"))]
+        {
+            let execution = self.execution.clone();
+            let payload = self.current_payload.clone();
+            let slots = slots.to_owned();
+
+            tokio::task::block_in_place(move || {
+                tokio::runtime::Handle::current().block_on(execution.get_account(
+                    &address,
+                    Some(&slots),
+                    &payload,
+                ))
+            })
+        }
 
-        // TODO: The above is the original implementation, but it can not be used
-        // in WASM environments.
-        //
-        // We are also not sure if this will ever be triggered since batch_fetch_accounts
-        // is pre-fetching the accounts.
-        panic!("not supported in wasm");
+        #[cfg(target_arch = "wasm32")]
+        {
+            let _ = address;
+            let _ = slots;
+            panic!("not supported in wasm");
```

Comment:
if we gonna need it, can we still have `TODO` there?

Reply:
added a TODO comment